src/Discord.Net.Core/Entities/Permissions/ChannelPermissions.cs

```diff
@@ -73,7 +73,7 @@ public static ChannelPermissions All(IChannel channel)
         public bool UseVAD => Permissions.GetValue(RawValue, ChannelPermission.UseVAD);
 
         /// <summary> If True, a user may adjust permissions. This also implictly grants all other permissions. </summary>
-        public bool ManagePermissions => Permissions.GetValue(RawValue, ChannelPermission.ManagePermissions);
+        public bool ManagePermissions => Permissions.GetValue(RawValue, ChannelPermission.ManageRoles);
```

Comment:
Added in 236775c2d8aca9481d6c51c674f5727f97adec04 and 9f1c6d0f425bc93b153f32f6e5b147f79d772499